identity_presentation/src/commonMain/kotlin/net/thechance/mena/identity/presentation/feature/profileFlow/changePassword/ChangePasswordScreen.kt

```diff
@@ -19,23 +19,23 @@ import mena.identity_presentation.generated.resources.profile_change_password
 import net.thechance.mena.designsystem.presentation.component.scaffold.Scaffold
 import net.thechance.mena.designsystem.presentation.theme.theme.MenaTheme
 import net.thechance.mena.designsystem.presentation.theme.theme.Theme
-import net.thechance.mena.identity.presentation.base.BaseScreen
-import net.thechance.mena.identity.presentation.components.AuthAppBar
-import net.thechance.mena.identity.presentation.components.ErrorSnackBar
-import net.thechance.mena.identity.presentation.screen.changePassword.ChangePasswordScreenUIEffect.NavigateBack
-import net.thechance.mena.identity.presentation.screen.changePassword.components.CurrentPasswordContent
-import net.thechance.mena.identity.presentation.screen.changePassword.components.NewPasswordContent
-import net.thechance.mena.identity.presentation.screen.profile.SnackBarUiState
+import net.thechance.mena.identity.presentation.core.base.BaseScreen
+import net.thechance.mena.identity.presentation.core.components.AuthAppBar
+import net.thechance.mena.identity.presentation.core.components.ErrorSnackBar
+import net.thechance.mena.identity.presentation.feature.profileFlow.changePassword.ChangePasswordScreenUIEffect.NavigateBack
+import net.thechance.mena.identity.presentation.feature.profileFlow.changePassword.components.CurrentPasswordContent
+import net.thechance.mena.identity.presentation.feature.profileFlow.changePassword.components.NewPasswordContent
+import net.thechance.mena.identity.presentation.feature.profileFlow.profile.SnackBarUiState
 import org.jetbrains.compose.resources.stringResource
 import org.jetbrains.compose.ui.tooling.preview.Preview
 
 
 class ChangePasswordScreen(
     val onSuccess: (SnackBarUiState?) -> Unit,
 ) : BaseScreen<ChangePasswordScreenViewModel,
-            ChangePasswordScreenUIState,
-            ChangePasswordScreenUIEffect,
-            ChangePasswordScreenInteractionListener>() {
+        ChangePasswordScreenUIState,
+        ChangePasswordScreenUIEffect,
+        ChangePasswordScreenInteractionListener>() {
 
```

Comment:
for consistency 
idk why but we use it in every screen 
```suggestion
class ChangePasswordScreen(
    val onSuccess: (SnackBarUiState?) -> Unit,
) : BaseScreen<
            ChangePasswordScreenViewModel,
            ChangePasswordScreenUIState,
            ChangePasswordScreenUIEffect,
            ChangePasswordScreenInteractionListener>() {
```